Data/Primitive/Array.hs

```diff
@@ -63,6 +63,10 @@ import Data.Functor.Identity
 
 import Text.ParserCombinators.ReadP
 
+#if MIN_VERSION_base(4,9,0) || MIN_VERSION_transformers(0,4,0)
+import Data.Functor.Classes (Eq1(..),Ord1(..),Show1(..))
```

Comment:
This leaves out `Read1`... was this intentional?

Reply:
I'll add this.